### DISCUSSION TITLE
It's possibile to deploy a static Flutter web application on the server?

### DISCUSSION THREAD

**Discussion by CesareIsHere** [serverpod/serverpod#634] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
Yes, I want do deploy my flutter web application on serverpod server, but I don't know how to do it.

**Comment by vlidholt** [comment, marked as answer]
Yes! One example is https://pixorama.live check the source code for how to do it here: https://github.com/serverpod/pixorama#hosting-the-flutter-app-with-serverpod

**Reply by CesareIsHere** [reply]
So it's necessary to put "build web" result into "web" folder of serverpod_server and than set the correct folder/source name in the static routes, right?

**Reply by vlidholt** [reply]
Yes, that sounds right.

**Reply by CesareIsHere** [reply]
Perfect!! Thank you so much for your help and quick response!
